Change the patch from "disabled" to "okay" to enable UART logging for the right joycon
Note: will conflict with the Joy-Con driver
--- a/kernel-dts/tegra210-abca-common.dtsi	2019-03-21 15:17:55.000000000 +0100
+++ b/kernel-dts/tegra210-abca-common.dtsi	2019-03-23 16:49:44.896298374 +0100
@@ -395,7 +395,7 @@
 	/* right joycon */
 	serial@70006040 {
 		compatible = "nvidia,tegra20-uart";
-		status = "disabled";
+		status = "okay";
 	};
 	serial@70006300 {
 		dma-names = "tx";